server/src/main/scala/org/http4s/server/logging/ServerLogging.scala

```diff
@@ -23,21 +23,24 @@ object ServerLogging {
   }
 
   def commonLogFormat[F[_]](clock: Clock = Clock.systemDefaultZone()): ServerLogging[F] = {
+    val log = org.log4s.getLogger
```

Comment:
I suggested either this or a `String => Unit`.  I'm now thinking maybe the latter makes more sense.  Then you could log directly to the console (and not have to do the irritating XML config) or go through log4s (and get nice rolling logs if you're not externally slurping stdout).

Reply:
commonLogFormat now takes an extra argument to allow custom output methods.